sdk/firstload/src/index.tsx

```diff
@@ -14,21 +14,24 @@ import {
 	Metric,
 	OnHighlightReadyOptions,
 	SessionDetails,
-} from '@highlight-run/client/src/types/types'
-import { MixpanelAPI, setupMixpanelIntegration } from './integrations/mixpanel'
+} from '@highlight-run/client/src/types/types.js'
+import {
+	MixpanelAPI,
+	setupMixpanelIntegration,
+} from './integrations/mixpanel.js'
 
-import { FirstLoadListeners } from '@highlight-run/client/src/listeners/first-load-listeners'
-import { GenerateSecureID } from '@highlight-run/client/src/utils/secure-id'
-import { HighlightSegmentMiddleware } from './integrations/segment'
-import configureElectronHighlight from './environments/electron'
+import { FirstLoadListeners } from '@highlight-run/client/src/listeners/first-load-listeners.js'
+import { GenerateSecureID } from '@highlight-run/client/src/utils/secure-id.js'
+import { HighlightSegmentMiddleware } from './integrations/segment.js'
+import configureElectronHighlight from './environments/electron.js'
 import firstloadVersion from './__generated/version'
 import {
 	SessionData,
 	getPreviousSessionData,
-} from '@highlight-run/client/src/utils/sessionStorage/highlightSession'
+} from '@highlight-run/client/src/utils/sessionStorage/highlightSession.js'
```

Comment:
is there is no way to set something in our tsconfig to do this across the entire package?

Reply:
Stackoverflow indicates that it's either done in-code or by regexp replacing:

https://stackoverflow.com/questions/62619058/appending-js-extension-on-relative-import-statements-during-typescript-compilat